release-notes.md

```diff
@@ -0,0 +1,106 @@
+[//]: # (<details>)
+
+[//]: # (<summary>Release Notes v1.0.1</summary>)
+
+[//]: # (<!-- always an empty line before table -->)
+
+## Secure API Gateway
+
+The ForgeRock Secure API Gateway for Open Banking is an open-source solution that helps financial services organizations deliver both test and production Open Banking and custom APIs from one framework, enforced by the ForgeRock Identity Platform.
+
+> These are the release notes for `Secure API Gateway` version `v1.0.1` named `Aerosmith`
+
+### Previous version
+- `v1.0.1`
+
+### Component Helm chart versions used by this release
+| component                                                                                             | Helm chart version | Full changelog                                                                                                       |
+|-------------------------------------------------------------------------------------------------------|--------------------|----------------------------------------------------------------------------------------------------------------------|
+| IG [secure-api-gateway-ob-uk](https://github.com/SecureApiGateway/secure-api-gateway-ob-uk)           | `1.0.1`            | [compare v1.0.0 vs v1.0.1](https://github.com/SecureApiGateway/secure-api-gateway-ob-uk/compare/v1.0.0...v1.0.1)     |
+| RS [secure-api-gateway-ob-uk-rs](https://github.com/SecureApiGateway/secure-api-gateway-ob-uk-rs)     | `1.0.1`            | [compare v1.0.0 vs v1.0.1](https://github.com/SecureApiGateway/secure-api-gateway-ob-uk-rs/compare/v1.0.0...v1.0.1)  |
+| RCS [secure-api-gateway-ob-uk-rcs](https://github.com/SecureApiGateway/secure-api-gateway-ob-uk-rcs)  | `1.0.1`            | [compare v1.0.0 vs v1.0.1](https://github.com/SecureApiGateway/secure-api-gateway-ob-uk-rcs/compare/v1.0.0...v1.0.1) |
+| RCS-UI [secure-api-gateway-ob-uk-ui](https://github.com/SecureApiGateway/secure-api-gateway-ob-uk-ui) | `1.0.2`            | [compare v1.0.1 vs v1.0.2](https://github.com/SecureApiGateway/secure-api-gateway-ob-uk-ui/compare/v1.0.1...v1.0.2)  | 
+
+### Release Notes v1.0.1
+
+### Upgrade Steps
+[n/a]
+
+### Breaking Changes
+[n/a]
+
+### New Features
+#### Addressed Issues
```

Comment:
Remove the Addressed Issues sub heading, I think the New Features heading is sufficient

Reply:
done